src/main/kotlin/io/zerobase/smarttracing/resources/AmazonSES.kt

```diff
@@ -0,0 +1,24 @@
+package io.zerobase.smarttracing.resources
+
+import software.amazon.awssdk.core.exception.SdkException
+import software.amazon.awssdk.regions.Region
+import software.amazon.awssdk.services.ses.SesClient
+import software.amazon.awssdk.core.SdkBytes
+import software.amazon.awssdk.services.ses.model.SendRawEmailRequest
+import software.amazon.awssdk.services.ses.model.RawMessage
+
+class AmazonSES : EmailSender {
+    private val client: SesClient
+
+    init {
+        client = SesClient.builder().region(Region.US_EAST_1).build()
+    }
```

Comment:
you are still missing the point on the dependency inversion principle. classes should *not* create what they need, they should be given it. do it like this:

```kotlin
class AmazonEmailSender(private val client: SesClient) {
...
}